public/css/module.less

```diff
@@ -1541,8 +1554,8 @@ input[type=submit].icon-button {
 
 }
 
-.breadcrumb li:not(:last-child) a:hover { background: @text-color; color: white; }
-.breadcrumb li:not(:last-child) a:hover:after { border-left-color: @text-color; }
+.breadcrumb li:not(:last-child) a:hover { .var(background, text-color); .var(color, "white"); }
```

Comment:
The :hover effect for the breadcrumb looks broken. It would be nice, if you found a way to make it look like this:

![Screen Shot 2021-11-23 at 17 47 42](https://user-images.githubusercontent.com/6977434/143067770-dec64f76-8f28-459a-b8a4-5cc4fbbe346b.png)

Reply:
Sadly, it looks like this. I just can't manage to fade the `border-left-color` the same way.

<img width="628" alt="Bildschirmfoto 2021-11-24 um 09 42 25" src="https://user-images.githubusercontent.com/57616252/143204011-bc406cc2-a593-4ae4-b622-ef77d41afc44.png">